luxonis_ml/data/augmentations/utils.py

```diff
@@ -173,32 +173,37 @@ def __call__(
             keypoints_visibility_batch.append(keypoints_visibility)
             keypoints_classes_batch.append(keypoints_classes)
 
-        # Apply transforms
-        # NOTE: All keys (including label_fields) must have _batch suffix when using BatchCompose
-        transformed = self.batch_transform(
-            image_batch=image_batch,
-            mask_batch=mask_batch,
-            bboxes_batch=bboxes_batch,
-            bboxes_visibility_batch=bboxes_visibility_batch,
-            bboxes_classes_batch=bboxes_classes_batch,
-            keypoints_batch=keypoints_batch,
-            keypoints_visibility_batch=keypoints_visibility_batch,
-            keypoints_classes_batch=keypoints_classes_batch,
-        )
+        transform_args = {
+            "image_batch": image_batch,
+            "bboxes_batch": bboxes_batch,
+            "bboxes_visibility_batch": bboxes_visibility_batch,
+            "bboxes_classes_batch": bboxes_classes_batch,
+            "keypoints_batch": keypoints_batch,
+            "keypoints_visibility_batch": keypoints_visibility_batch,
+            "keypoints_classes_batch": keypoints_classes_batch,
+        }
+        if return_mask:
```

Comment:
Do we see any concerns with mask_batch missing downstream? Would it be safer to keep it in dict but have it as an empty value?

Reply:
No, the augmentations will still work if mask_batch is simply missing. I think it does not pose any threat.

 Moreover, using dummy keypoints (just one keypoint at x:0, y:0) or bounding boxes does not affect the time it takes to apply the augmentations.

Reply:
CC: @JSabadin